### ISSUE TITLE
How to Acquire Moon stones?

### ISSUE THREAD

**Issue by Lion4K** [rwtema/extrautilities#1041]
JEI does not shows an Recipe, nor does a description (usually with non craft able items)

Appreciates anyone that can tell me the recipe.

**Comment by iKurama** [comment]
It can be found as an ore.

**Comment by Lion4K** [comment]
But there is no Moon Stone ore either

**Comment by iKurama** [comment]
It will spawn. Stone texture with a slightly darker circle-ish in the middle.

**Comment by b0bst3r** [comment]
It's way too rare to say it's in a glass recipe and isn't mined in the EU mining dimension either, we found a total of 2 moonstone in about 16 million blocks mined.

**Comment by Lion4K** [comment]
yeah. but the thing is, its ore block is not shown in JEI, does anyone know the ID?
